src/donut/dbxray/generate/datapotato.cljc

```diff
@@ -0,0 +1,43 @@
+(ns donut.dbxray.generate.datapotato
+  (:require
+   [clojure.string :as string]
+   [donut.dbxray.generate :as ddg]))
+
+(defn- gen-table-prefix
+  [table-name]
+  (let [words (string/split (name table-name) #"_")
+        initials (map (comp str first) words)]
+    {:prefix (keyword (apply str initials))}))
+
+(defn- add-table-prefix-to-col-name
+  [table-name col-name]
+  (let [table (name table-name)]
+    (keyword (str table "/" (name col-name)))))
+
+(defn- gen-relations-for-column
```

Comment:
hmmm what do you think of taking a different approach where we have a function `table-relations` that produces `{:relations ...}` if there are any relations? That way in `gen-potato-for-table` we could have `(merge prefix (table-relations table-name columns)` instead of `reduce` + `merge-with into`. I like having the responsibility for producing the relations in one function instead of spread across two.